src/utils.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
+import type { LocaleType } from '@paypal/sdk-constants/src/';
+import { COUNTRY_LANGS, LANG } from '@paypal/sdk-constants/src';
+
+
+export function getComputedLocales(locale : string) : LocaleType {
+    let [ lang, country ] = locale.split('_');
+    // $FlowFixMe
+    const countryLangs = COUNTRY_LANGS[country];
+
+    if (countryLangs.includes(LANG.ZH_HANT)) {
+        lang = LANG.ZH_HANT;
+    }
+
+
+    // $FlowFixMe
```

Comment:
I noticed you have two `$FlowFixMe` comments in here. What flow errors are you getting? It would be great to try and eliminate those and only use them as a last resort.

Reply:
I thought it would be more difficult to solve them but I found a way, thanks!!